src/go/k8s/pkg/resources/statefulset.go

```diff
@@ -208,18 +217,39 @@ func (r *StatefulSetResource) Obj() (k8sclient.Object, error) {
 					},
 					InitContainers: []corev1.Container{
 						{
-							Name:		configuratorContainerName,
-							Image:		r.pandaCluster.FullImageName(),
-							Command:	[]string{"/bin/sh", "-c"},
-							Args:		[]string{configuratorPath},
+							Name:			configuratorContainerName,
+							Image:			"gcr.io/vectorized/configurator:latest",
```

Comment:
What's the plan for providing the image name and in particular the version? We cannot leave it as "latest". This is related to the upgrade later on.

Reply:
The plan is to version configurator. Similarly to the operator. 

There are 2 options:

- ControllerManagerConfig that have configurator image and version
- At the build time using the `ldflags` provide docker tag of the configurator

I will implement the later as simpler solution

Reply:
should we just for simplicity version both the same so 0.1.1 of operator will use 0.1.1 of configurator?

Reply:
Agree, but currently we don't settle on versioning scheme. https://github.com/vectorizedio/redpanda/issues/690